src/main/AchievementData/Achievements.txt

```diff
@@ -0,0 +1,4 @@
+0
```

Comment:
Bu dosyanın yeri doğru değil. Bu dosya bir kod dosyası olmadığı için resources altına bir yere koyman daha doğru.